forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -310,9 +310,13 @@ def acquire_dfe_data(qc: QuantumComputer, expr: TomographyExperiment, n_shots=10
     if mitigate_readout_errors:
         res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset))
     else:
-        res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset, readout_symmetrize=None,
+        res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset, symmetrize_readout=None,
                                        calibrate_readout=None))
 
+    # identify the qubits being measured
+    max_weight_op = _max_weight_operator([e[0].out_operator for e in expr])
```

Comment:
I feel a little uncomfortable relying on a private method of the `pyquil.operator_estimation` module. Why not instead try:
```python
qubits = list(set(e[0].out_operator.get_qubits() for e in expr))
```

Reply:
Sure, that makes sense. Updated.